ruby/files_and_serialization/project_event_manager.md

```diff
@@ -11,12 +12,13 @@ This section contains a general overview of topics that you will learn in this l
 - Utilize the data to contact a remote service.
 - Populate a template with user data.
 - Manipulate [strings](https://docs.ruby-lang.org/en/3.2/String.html).
-- Access [Google's Civic Information API](https://developers.google.com/civic-information/) through
-  the [Google API Client Gem](https://github.com/google/google-api-ruby-client).
+- Access [Google's Civic Information API](https://developers.google.com/civic-information/) through the [Google API Client Gem](https://github.com/google/google-api-ruby-client).
 - Use [ERB](https://docs.ruby-lang.org/en/3.2/ERB.html) (Embedded Ruby) for templating.
 
 <div class="lesson-note" markdown="1">
-  This tutorial is open source. If you notice errors, typos, or have questions/suggestions, please [submit them to the project on GitHub](https://github.com/TheOdinProject/curriculum/blob/main/ruby/files_and_serialization/project_event_manager.md).
+
+  This tutorial is open source. If you notice errors, typos, or have questions/suggestions, please [open an issue on GitHub](https://github.com/TheOdinProject/curriculum/blob/main/ruby/files_and_serialization/project_event_manager.md).
+
 </div>
```

Comment:
To fix this lint error, we can place this lesson note in the Introduction section (where it fits better anyway IMO).